packages/rpc/rpc.1.9.51/opam

```diff
@@ -23,7 +21,7 @@ remove: [
 depends: [
   "ocaml" {< "4.06.0"}
   "oasis" {build}
-  "cppo" {build}
+  "cppo" {build & < "1.5.0"} | "cppo_ocamlbuild" {build & "cppo" >= "1.5.0"}
```

Comment:
@mseri this doesn't work (`"cppo" >= "1.5.0"` is not a valid attribute). Why not just ask for `cppo_ocamlbuild`? There is no packages in opam-repository right now that requires `cppo < 1.6.0`. Except being (too?) precise, it doesn't add any benefits imo.